src/test/java/org/apache/commons/cli/PatternOptionBuilderTest.java

```diff
@@ -159,13 +161,12 @@ public void testURLPattern() throws Exception
     @Test
     public void testExistingFilePattern() throws Exception
     {
-        final Options options = PatternOptionBuilder.parsePattern("f<");
+        final Options options = PatternOptionBuilder.parsePattern("f<g<");
         final CommandLineParser parser = new PosixParser();
-        final CommandLine line = parser.parse(options, new String[] { "-f", "test.properties" });
-
-        assertEquals("f value", new File("test.properties"), line.getOptionObject("f"));
-
-        // todo test if an error is returned if the file doesn't exists (when it's implemented)
+        final CommandLine line = parser.parse(options, new String[] { "-f", "test.properties", "-g", "/dev/null" });
+        
+        assertNotNull("option g not parsed, or not FileInputStream", (FileInputStream) line.getOptionObject("g"));
+        assertNull("option f parsed", (FileInputStream) line.getOptionObject("f"));
```

Comment:
> Yes, It is null because test.properties does not exist, or not readable. This can be made clearer somehow.

Maybe by calling it `non-existing.file`?

> Another task is to find a cross-platform file which is exist (and not /dev/null).

You could put add dummy file in `src/test/resources` and use that one.